homeassistant/components/nederlandse_spoorwegen/strings.json

```
@@ -0,0 +1,144 @@
+        "title": "Nederlandse Spoorwegen setup",
+        "description": "Set up your Nederlandse Spoorwegen integration.",
+        "data": {
+          "api_key": "[%key:common::config_flow::data::api_key%]"
+        },
+        "data_description": {
+          "api_key": "Your NS API key."
+        }
+      },
+      "routes": {
+        "title": "Add route",
+        "description": "Add a train route to monitor. [Find Dutch station codes here]({station_list_url}). Time is optional and must be in HH:MM:SS format (e.g., 08:06:00). If omitted, the next available train will be shown.",
+        "data": {
+          "name": "Route name",
+          "from": "From station",
+          "to": "To station",
+          "via": "Via station (optional)",
+          "time": "Departure time (optional)"
+        },
+        "data_description": {
+          "name": "A name for this route.",
+          "from": "Departure station code (e.g., AMS)",
+          "to": "Arrival station code (e.g., UTR)",
+          "via": "Optional via station code (e.g., RTD)",
+          "time": "Optional departure time in HH:MM:SS (24h)"
+        }
+      },
+      "reauth": {
+        "title": "Re-authenticate",
+        "description": "Your NS API key needs to be updated.",
+        "data": {
+          "api_key": "[%key:common::config_flow::data::api_key%]"
+        },
+        "data_description": {
+          "api_key": "Enter your new NS API key."
+        }
+      },
+      "reconfigure": {
+        "title": "Reconfigure",
+        "description": "Update your NS API key.",
+        "data": {
+          "api_key": "[%key:common::config_flow::data::api_key%]"
+        },
+        "data_description": {
+          "api_key": "Enter your new NS API key."
+        }
+      }
+    },
+    "error": {
+      "cannot_connect": "Could not connect to NS API. Check your API key.",
+      "invalid_auth": "Invalid API key.",
+      "unknown": "[%key:common::config_flow::error::unknown%]"
+    },
+    "abort": {
+      "already_configured": "This API key is already configured.",
+      "reauth_successful": "Re-authentication successful.",
+      "reconfigure_successful": "Reconfiguration successful."
+    }
+  },
+  "options": {
+    "step": {
+      "init": {
+        "title": "Configure routes",
+        "description": "Configure the routes.",
+        "data": {
+          "action": "Action"
+        },
+        "data_description": {
+          "action": "Edit, add, or delete train routes for this integration."
+        }
+      },
+      "select_route": {
+        "title": "Select route",
+        "description": "Choose a route to {action}.",
+        "data": {
+          "route_idx": "Route"
+        },
+        "data_description": {
+          "route_idx": "Select the route you want to edit or delete."
+        }
+      },
+      "add_route": {
+        "title": "Add route",
+        "description": "[%key:component::nederlandse_spoorwegen::config::step::routes::title%]",
```

Looks like `title` and `description` got swapped here with the single reference, and we can reference both, too:

```suggestion
        "title": "[%key:component::nederlandse_spoorwegen::config::step::routes::title%]",
        "description": "[%key:component::nederlandse_spoorwegen::config::step::routes::description%]",
```